spec/policies/topic_request_policy_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe TopicRequestPolicy do
+  let(:topic_request) { FactoryGirl.build(:topic_request) }
+  context 'the person is not signed in' do
+    subject { described_class.new(nil_person, topic_request) }
+    let(:nil_person) { nil }
+
+    describe '#create?' do
```

Comment:
Point taken. This code has all been removed along with the policy it was testing though.